src/App.js

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+
+import UserProfile from "./components/User/UserProfile";
+import Photos from "./components/gallery/Photos";
+import "./App.css";
+
+function App() {
+
+  const [data, setData] = useState(null);
+
+  useEffect(() => {
+     
+      let myHeaders = new Headers();
+      myHeaders.append("Origin", "https://localhost.com");
```

Comment:
we hardcoded it before, but better not do that.
try this on ur chrome dev tools js console 
`window.location.origin`
or `window.location.hostname`

ull see what i mean ;)

Reply:
hmm...
so what's a fix for this?